src/components/Container/Container.jsx

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+
+const boundClassNames = lucidClassNames.bind('&-Container');
+
+const {
+	node,
+	string
+} = React.PropTypes;
+
+function getHeader(headerChildProp, title){
```

Comment:
This is basically just a default value for the header, I'd scrap these functions